alea/src/variance.cpp

```diff
@@ -171,10 +192,10 @@ void var_result<T,Str>::reduce(const reducer &r, bool pre_commit, bool post_comm
     internal::check_valid(*this);
     if (pre_commit) {
         store_->convert_to_sum();
-        r.reduce(sink<T>(store_->data().data(), store_->data().rows()));
-        r.reduce(sink<var_type>(store_->data2().data(), store_->data2().rows()));
-        r.reduce(sink<double>(&store_->count(), 1));
-        r.reduce(sink<double>(&store_->count2(), 1));
+        r.reduce(view<T>(store_->data().data(), store_->data().rows()));
+        r.reduce(view<var_type>(store_->data2().data(), store_->data2().rows()));
+        r.reduce(view<double>(&store_->count(), 1));
```

Comment:
count is double? :hurtrealbad:

Reply:
Well, yes, for consistency with count2_, as the sum of squares can easily overflow a long.

Reply:
because in other places count is size_t